application/gui/input_dialog.py

```diff
@@ -17,6 +17,8 @@ class OverwriteOption:
 	NEW_NAME = 1
 	APPLY_TO_ALL = 2
 
+# header bars are only available in GTK 3.12 and up
+_use_header_bar = { 'use_header_bar': True } if hasattr(Gtk.Dialog, 'get_header_bar') else {}
```

Comment:
Move this to `InputDialog` as class property and make it "public" (`use_header_bar`). This way it will make sense to use it in all dialogs. If dialog is not inheriting `InputDialog` it's still simple enough to use `InputDialog.use_header_bar` as it's class property.